tests/TestFixtures/TNSTestNativeCallbacks.h

```diff
@@ -12,6 +12,10 @@
 
 @interface TNSTestNativeCallbacks : NSObject
 
+typedef void (^TNSBlock)();
+
+@property(assign, nonatomic) TNSBlock block;
```

Comment:
> imo JS is brittle.

FTFY :grinning: 

We can add an expectation that the `block` property exists with the `in` operator, but what you've touched upon is a problem everywhere, not just in this test.